moto/sagemaker/models.py

```diff
@@ -3,7 +3,7 @@
 import random
 import string
 from datetime import datetime
-from typing import Any, Dict, List, Optional, Iterable, Union
+from typing import Any, Dict, List, Literal, Optional, Iterable, Union
```

Comment:
Apparently this import is unused - that's why the linter is failing:

https://github.com/getmoto/moto/actions/runs/5457165070/jobs/9931131021?pr=6454

Reply:
https://github.com/getmoto/moto/pull/6454/commits/ff3e37c54cc7877a0102df9c6d7681844177397a